install

```diff
@@ -237,8 +241,8 @@ unless mac?
   if File.writable?(HOMEBREW_PREFIX_DEFAULT) || File.writable?("/home/linuxbrew") || File.writable?("/home")
     HOMEBREW_PREFIX = HOMEBREW_PREFIX_DEFAULT.freeze
   else
-    Kernel.system "/usr/bin/sudo -n -v 2>/dev/null"
-    unless $CHILD_STATUS.success?
+    out = `/usr/bin/sudo -n -l mkdir 2>&1`
+    if !$CHILD_STATUS.success? && out == "sudo: a password is required\n"
```

Comment:
Could you please rename `out` to `sudo_output`?

Reply:
Renamed